ui/src/app/components/icons.tsx

```diff
@@ -17,8 +17,8 @@ export const styles = {
     height: '20px'
   },
 
-  warningIcon: {
-    color: colors.warning,
+  dangerIcon: {
+    color: colors.danger,
```

Comment:
As it turns out, `ValidationIcon` is only used in registration, so I'd rather change it one place than pass in styles in two places.